test/unit/context_adapter_test.js

```diff
@@ -0,0 +1,232 @@
+/*
+ * Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U
+ *
+ * This file is part of push-2-tweet
+ *
+ * push-2-tweet is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * push-2-tweet is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with push-2-tweet.
+ * If not, seehttp://www.gnu.org/licenses/.
+ *
+ * For those usages not covered by the GNU Affero General Public License
+ * please contact with::[german.torodelvalle@telefonica.com]
+ */
+
+'use strict';
+
+var push2Tweet = require('../../lib/push_2_tweet');
+var p2tConfig = require('../../lib/push_2_tweet_configuration');
+var testHelper = require('./context_adapter_test_helper');
```

Comment:
There are some references to "context_adapter_things" along the code, in names and comments. Please, search and replace them with "third_party_things" as it seems confusing for a first time read.